doc/source/cluster/examples/slurm-basic.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
I think this needs to be regenerated? not sure; seems to be a little out of sync w/ slurm-template.sh

Reply:
purposely, it's a separate script (not autogenerated)